web/src/styles/base/background-colors.scss

```diff
@@ -98,6 +98,10 @@
   background-color: rgba(255, 255, 255, 0) !important;
 }
 
+.bg-base-lighter {
+  background-color: $base-lighter !important;
+}
```

Comment:
doesn't this exist in NJWDS already? Or am I confused at the purpose here?

Reply:
Deleting the css.